components/common/PanelsCreator.tsx

```diff
@@ -64,7 +64,7 @@ export function PanelsCreator({
         <Card className={style.panelDiv}>
           <BodyText
             data-cy={`notAssigned${panelsName}`}
-          >{`You are not assigned to any ${panelsTitle}.`}</BodyText>
+          >{`You are not assigned to any p${panelsTitle.slice(1)}.`}</BodyText>
```

Comment:
Is the `p` in front of `${panelsTitle.slice(1)}` not added intentionally?

Reply:
@DorisWongNHS, it is intentional; inconsistent 'P' flagged-up so 'un-capitalised' it.